src/core_plugins/kbn_vislib_vis_types/public/controls/heatmap_options.html

```diff
@@ -0,0 +1,196 @@
+<div>
```

Comment:
i dont see it being reused but i wanted to split heatmap options down to smaller sections. in the future i plan to do similar things for other charts so controls here will be grouped by chart type.